policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/ns_basic_personal_amount/lower_income_threshold.yaml

```diff
@@ -0,0 +1,11 @@
+description: Nova Scotia lower income threshold for maximum basic personal amount in basic personal amount.
```

Comment:
Change to:
Nova Scotia limits this lower income threshold under the basic personal amount eligibility.